VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,254 @@
+
+First, a 2% tax is applied.
+* 2% * 1025.51020408 = 20.51020408 Atoms go to the reserve pool
+
+1005 Atoms now remain. Let’s assume that the proposer included 100% of the signatures in its block. It thus obtains the full bonus of 5%.
+
+We have to solve this simple equation to find the reward R for each validator:
+
+`9*R + R + R*5% = 1005 ⇔ R = 1005/10.05 = 100`
+
+* For the proposer validator
+  * The pool obtains R + R*5% = 105 Atoms
+  * Commission = 105*80%*1% = 0.84 Atoms
+  * Validator reward = 100 * 20% + Commission = 21.84 Atoms
+  * Delegators rewards = 105 * 80% - Commission = 83.16 Atoms (each delegator will be able to claim its portion of these rewards in proportion of their stake)
+* For each non-proposer validator
+  * The pool obtains R = 100 Atoms
+  * Commission = 100*80%*1% = 0.8 Atoms
+  * Validator reward = 100 * 20% + Commission = 20.8 Atoms
+  * Delegators rewards = 100 * 80% - Commission = 79.2 Atoms (each delegator will be able to claim its portion of these rewards in proportion of their stake)
+
+### What are the slashing conditions?
+
+If a validator misbehaves, its bonded stake along with its delegator’s stake and will be slashed. The severity of the punishment depends on the type of fault. There are 3 main faults that can result in loss of funds for a validator and its delegators :
+
+* **Double signing:** If someone reports on chain A that a validator signed two blocks at the same height on chain A and chain B, this validator will get slashed on chain A
+* **Unavailability:** If a validator’s signature has not been included in the last X blocks, the validator will get slashed by a marginal amount proportional to X. If X is above a certain limit Y, then the validator will get unbonded
+* **Non-voting:** If a validator did not vote on a proposal and once the fault is reported by a someone, its stake will receive a minor slash. 
+
+Note that even if a validator does not intentionally misbehave, it can still be slashed if its node crashes, looses connectivity, gets DDOSed, or if its private key is compromised.
+
+### Incentives for validator #101 to stake?
+
+Validator candidates that do not have enough total stake to be in the validator set will have to run validator operation (or at least be ready to) without getting any revenue. 
+
+There are solutions that are considered to mitigate this issue. For example, we could create a sub-layer of delegation. Validator candidate could delegate to existing validators while they’re trying to attract delegators, thereby generating revenue. At the same time, delegators already bonded to validators could signal their intention to switch to this validator candidate should it get enough stake to enter the validator set. When the validator candidate would get enough pledge, it would transition to a validator and all the delegator who pledged would join its staking pool.
```

Comment:
`Validator candidates ...`

`they're` and `they are`

`... get enough stake.`

Let's not use pledged. Let's stick to delegate.

Reply:
No to ` ... get enough stake`, it becomes confusing. I think the precision is important.

pledge seems fine to me